enh(release): Add Centreon Web 22.04.1 RN (#1682)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md

## Centreon Web

### 22.04.1

#### Enhancements

> - [Authentication] Allow to automatically import new users using OpenId Connect protocol

Reviewer: ```suggestion
- [Authentication] Added the permission to import automatically new users using the OpenId Connect protocol
```

> - [Authentication] Apply ACL Group(s) definition on login for OpenID Connect users

Reviewer: ```suggestion
- [Authentication] Applied an ACL Group(s) definition on login for OpenID Connect users
```

- [Configuration] Extended the size of the URL, Notes and Action URL fields to avoid truncating long URLs

> - [Core] Handle properly the switch between Resource Status repositories

Reviewer: ```suggestion
- [Core] Properly managed the switch between Resource Status repositories
```

- [Install] Improved error handling during installation

> - [UX] Improved OpenId Connect form

Reviewer: ```suggestion
- [UX] Improved the OpenId Connect form
```

#### Bug fixes

- [API] Fixed MBI APIs with latest version of Centreon

> - [Administration] Fixed display of the end date of the licenses

Reviewer: ```suggestion
- [Administration] Fixed the display of the end date of the licenses
```

> - [Administration] Fixed scroll when reduce screen size to access to all items

Reviewer: ```suggestion
- [Administration] Fixed the scrolling when reducing the screen size to access all items
```

- [Configuration] Fixed export when host group disabled

> - [Configuration] Fixed export when service group disabled

Reviewer: ```suggestion
- [Configuration] Fixed the export when the service group is disabled
```

> - [Configuration] Fixed export when service template disabled

Reviewer: ```suggestion
- [Configuration] Fixed the export when the service template is disabled
```

- [Configuration] Massive change failed to set contact/contactgroup additive inheritance

> - [Configuration] When a host receives no host template from the mappers, the default template that is hard-coded in the Plugin Pack will be applied to this host

Reviewer: Not sure to understand this one. What was the issue exactly?

> - [Core] Database partitioning fixed for MySQL 8

Reviewer: ```suggestion
- [Core] Fixed the database partitioning for MySQL 8
```

> - [Core] Fixed SQL queries when databases names contain dash

Reviewer: ```suggestion
- [Core] Fixed SQL queries when databases names contained a dash
```

- [Core] Re-added href in menus
- [Install] Fixed SQL issue during update
- [Install] Fixed rights on /usr/share/centreon/.env.local.php file for Debian package

> - [Install] Fixed waterfall visual effect in extension details tile

Reviewer: ```suggestion
- [Install] Fixed the waterfall visual effect in the extension's details tile
```

> - [Monitoring] Fixed notification number in legacy pages

Reviewer: 'the number of notifications' or 'the notification number'?

- [Monitoring] Fixed the "Last_update" column in legacy pages

> - [Resources Status] Fixed timeperiod group button and custom period selectors heights

Reviewer: ```suggestion
- [Resources Status] Fixed the timeperiod group button and custom period selectors heights
```

> - [UI] Close the menu when a navigation item is clicked

Reviewer: ```suggestion
- [UI] Now close the menu when a navigation item is clicked
```

> - [UI] Fixed display of CSS code with Firefox browser

Reviewer: ```suggestion
- [UI] Fixed the display of CSS code with Firefox browser
```

- [UI] Fixed header and skeleton UI instability

> - [UX] Reduce the timeout preventing from closing the menu unexpectedly

Reviewer: ```suggestion
- [UX] Reduced the timeout to prevent the menu from closing unexpectedly
```

> - [Widget] Use ACL to get list of poller to filter display of services

Reviewer: What was the issue exactly?

#### Security fixes

> - [Administration] Sanitize and bind ACL Group queries

Reviewer: ```suggestion
- [Administration] Sanitized and bound ACL Group queries
```

> - [Administration] Sanitize and bind ACL resources queries

Reviewer: ```suggestion
- [Administration] Sanitized and bound ACL resources queries
```

> - [Administration] Sanitize and bind escalation form queries

Reviewer: ```suggestion
- [Administration] Sanitized and bound escalation form queries
```

- [Configuration] Fixed SQLi vulnerability in escalation form
- [Configuration] Fixed XXS vulnerability in escalation form

> - [Configuration] Sanitize and bind "User" class queries

Reviewer: ```suggestion
- [Configuration] Sanitized and bound "User" class queries
```

> - [Configuration] Sanitize and bind "downtime" queries

Reviewer: ```suggestion
- [Configuration] Sanitized and bound "downtime" queries
```

> - [Configuration] Sanitize and bind "hostgroups" queries

Reviewer: ```suggestion
- [Configuration] Sanitized and bound "hostgroups" queries
```

> - [Configuration] Sanitize and bind "hosts" queries

Reviewer: ```suggestion
- [Configuration] Sanitized and bound "hosts" queries
```

> - [Configuration] Sanitize and bind "meta_service" related queries

Reviewer: ```suggestion
- [Configuration] Sanitized and bound "meta_service" related queries
```

> - [Configuration] Sanitize and bind "poller" queries

Reviewer: ```suggestion
- [Configuration] Sanitized and bound "poller" queries
```

> - [Configuration] Sanitize and bind "pollers" queries

Reviewer: ```suggestion
- [Configuration] Sanitized and bound "pollers" queries
```

> - [Configuration] Sanitize and bind contact form queries

Reviewer: ```suggestion
- [Configuration] Sanitized and bound contact form queries
```

> - [Configuration] Sanitize and bind timeperiod form queries

Reviewer: ```suggestion
- [Configuration] Sanitized and bound timeperiod form queries
```

- [Configuration] Sanitized and bound queries in virtual metrics configuration

> - [Core] Remove deprecated switch in encodePass() method

Reviewer: ```suggestion
- [Core] Removed deprecated switch in encodePass() method
```

> - [Core] Updated PHP libraries for security issue

Reviewer: ```suggestion
- [Core] Updated PHP libraries for security issues
```

Reviewer: Requested changes to be applied on the 2nd same file as well.

## Centreon Web

### 22.04.1

> 

Reviewer 2: ```suggestion

Release date: `July 28, 2022`

```

#### Enhancements

> - [Administration] Fixed consistency of ACLs with new poller creation wizard structure

Reviewer: ```suggestion
- [Administration] Added consistency in ACLs with the new structure of the poller creation wizard
```

#### Bug fixes

- [API] Fixed MBI APIs with the latest version of Centreon
- [Administration] Fixed the display of the end date of the licenses
- [Administration] Fixed the scrolling when reducing the screen size to access all items
- [Configuration] Fixed contact/contactgroup additive inheritance configuration using massive change
- [Configuration] Fixed empty host template from mappers (Host Discovery) by using default template form Plugin Packs discovery rule
- [Configuration] Fixed the export when the host group is disabled
- [Configuration] Fixed the export when the service group is disabled
- [Configuration] Fixed the export when the service template is disabled
- [Core] Fixed SQL queries when databases names contained a dash
- [Core] Fixed the database partitioning for MySQL 8

> - [Core] Re-added href link in menus

Reviewer: 'Fixed href on links that were broken in menus'?

- [Install] Fixed an SQL issue during update
- [Install] Fixed rights on the /usr/share/centreon/.env.local.php file for Debian package
- [Install] Fixed the waterfall visual effect in the extension's details tile
- [Monitoring] Fixed the "Last_update" column in legacy pages
- [Monitoring] Fixed the notification number in legacy pages
- [Resources Status] Fixed the timeperiod group button and custom period selectors heights
- [UI] Fixed header and skeleton UI instability
- [UI] Fixed the display of CSS code with Firefox browser
- [UI] Now close the menu when a navigation item is clicked
- [UX] Reduced the timeout to prevent the menu from closing unexpectedly

> - [Widget] Use ACL to get list of poller in widget configuration to filter display of services

Reviewer: Still unable to understand the fixed issue here
